src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
`gettype(1) === "integer"`

I am not aware of any standard like this. And again, it is documentation, not code.